### PR TITLE
Simplify and Update RPI.edu.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/RPI.edu.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		SSL peer certificate was not OK:
+			 - bookstore.rpi.edu
+			 - documents.studentsenate.rpi.edu
+
+		Peer certificate cannot be authenticated with given CA certificates:
+			 - finance.rpi.edu
+			 - hr.rpi.edu
+			 - srfs.rpi.edu
+
+		Secure connection redirects to plaintext:
```
Comment (J0WI) [inline src/chrome/content/rules/RPI.edu.xml]: Could not confirm, but MCB is triggered.

File: src/chrome/content/rules/RPI.edu.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+			 - events.rpi.edu
+
+		Mixed content blocking (MCB) triggered:
```
Comment (J0WI) [inline src/chrome/content/rules/RPI.edu.xml]: Maybe it's worth to create an additional `platform="mixedcontent"` rule?

**Comment by cschanaj** [comment]
@J0WI done. thanks!

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**